packages/shared/src/styles/devices.scss

```diff
@@ -94,3 +94,15 @@ $min-desktop-width: 1280px;
         @content;
     }
 }
+
+@mixin tablet-portrait-screen {
+    @media (min-width: #{$max-mobile-width + 1}) and (max-width: #{$min-desktop-width - 1}) and (orientation: portrait) {
```

Comment:
hi, these calculations for?

Reply:
```
$max-mobile-width: 600px;
$min-desktop-width: 1280px;
```

These variables define the maximum width for mobile devices and the minimum width for desktop devices. We need to create a media query for tablet devices that fall within this range.